CMake/CMakeLibs.cmake

```diff
@@ -68,50 +68,64 @@ macro(ADD_OSQUERY_LINK IS_CORE LINK)
   endif()
 endmacro(ADD_OSQUERY_LINK)
 
-macro(ADD_OSQUERY_LINK_INTERNAL LINK LINK_PATHS LINK_SET)
-  set(LINK_PATHS_RELATIVE
-    "${BUILD_DEPS}/lib"
-    "${CMAKE_BUILD_DIR}/third-party/*/lib"
-    ${LINK_PATHS}
+if(WIN32)
```

Comment:
I don't like this logic existing in `CMakeLibs`, if there's non-macro code, we should bias towards the root `CMakeLists.txt`. But make sure the `OS_LIB_DIRS` variable is set before including `CMakeLibs` for clarity.